statechart/models/interpreter.py

```diff
@@ -7,15 +7,12 @@
 from sismic.interpreter import Interpreter as SismicInterpreter
 from sismic.model import Event
 
-from odoo.exceptions import except_orm
-
 
 def _root_cause(e):
-    if isinstance(e, except_orm):
-        return e
-    if not hasattr(e, "__cause__") or not e.__cause__:
-        return e
-    return _root_cause(e.__cause__)
+    cause_exc = getattr(e, "__cause__", None)
+    if cause_exc:
+        return cause_exc
+    return e
```

Comment:
```suggestion
    return getattr(e, "__cause__", e)
```

This could be simplified like this, or even inlined at the two places where this is used, since the `_root_cause` name does not meany much anymore.

Reply:
Hm, no. I need more coffee :)

Reply:
Rather, `(e.__cause__ or e)`, because `__cause__` is part of `BaseException` so getattr should not be necessary anymore.